image/cinc/install_ubuntu_pkg.sh

```diff
@@ -41,7 +41,7 @@ apt install -yq --no-install-recommends \
   python3-pip \
   python3-psutil \
   python3-six \
-  resource-agents \
+  resource-agents* \
```

Comment:
Since this is expected to work with the latest ubuntu:rolling isn't it better to be explicit and list the exact packages that replace `resource-agents`?

Reply:
Our CI is using the rolling release, but users can build on any of their liking by overriding the ```OS_IMAGE```.

Reply:
OK, that makes sense, it's potentially a bit better for some users.  Thanks!